.github/workflows/Bonsai.yml

```diff
@@ -0,0 +1,460 @@
+        if: matrix.collect-packages && steps.pack.outcome == 'success' && always()
+        with:
+          name: Packages${{matrix.artifacts-suffix}}
+          if-no-files-found: error
+          path: artifacts/package/${{matrix.configuration-lower}}/**
+
+      - name: Collect portable zip
+        uses: actions/upload-artifact@v4
+        if: steps.create-portable-zip.outcome == 'success' && always()
+        with:
+          name: PortableZip${{matrix.artifacts-suffix}}
+          if-no-files-found: error
+          path: artifacts/Bonsai.zip
+
+      - name: Collect installer
+        uses: actions/upload-artifact@v4
+        if: steps.create-installer.outcome == 'success' && always()
+        with:
+          name: Installer${{matrix.artifacts-suffix}}
+          if-no-files-found: error
+          path: artifacts/bin/Bonsai.Setup.Bootstrapper/${{matrix.configuration-lower}}-x86/**
+
+  # =====================================================================================================================================================================
+  # Determine which packages need to be published
+  # =====================================================================================================================================================================
+  determine-changed-packages:
+    name: Detect changed packages
+    runs-on: ubuntu-latest
+    # We technically only need the dummy build jobs, but GitHub Actions lacks the ability to depend on specific jobs in a matrix
+    needs: build-and-test
+    if: github.event_name != 'pull_request'
+    steps:
+      # ----------------------------------------------------------------------- Checkout
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      # ----------------------------------------------------------------------- Setup tools
+      - name: Setup Python 3.11
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.11'
+
+      # ----------------------------------------------------------------------- Download packages for comparison
+      - name: Download packages for comparison
+        uses: actions/download-artifact@v4
+        with:
+          pattern: Packages*
+          path: artifacts
+
+      # ----------------------------------------------------------------------- Compare packages
+      - name: Compare packages
+        id: compare-packages
+        run: python .github/workflows/compare-nuget-packages.py artifacts/Packages-dummy-prev/ artifacts/Packages-dummy-next/ artifacts/Packages/ artifacts/ReleaseManifest
+
+      # ----------------------------------------------------------------------- Collect release manifest
+      - name: Collect release manifest
+        uses: actions/upload-artifact@v4
+        if: steps.compare-packages.outcome != 'skipped' && always()
+        with:
+          name: ReleaseManifest
+          if-no-files-found: error
+          path: artifacts/ReleaseManifest
```

Comment:
I would worry that shuttling these into the same folder like that might imply they're related somehow (which they really aren't) or they're created by the same tool (which again they aren't.)

Maybe a good way to look at these files is that they're build artifacts that defy categorization.

These files really only ever exist on a filesystem that humans never really interact with.

When someone does interact with these files (in the context of a filesystem), it's because they're testing the CI scripts locally in which case it's just gonna use whatever location they specify.